src/entity/Living.php

```diff
@@ -503,7 +503,7 @@ protected function doHitAnimation() : void{
 		$this->broadcastAnimation(new HurtAnimation($this));
 	}
 
-	public function knockBack(float $x, float $z, float $base = 0.4) : void{
+	public function knockBack(float $x, float $z, float $base = 0.4, bool $vanilla = false) : void{
```

Comment:
This is unnecessarily limiting. A vertical limit parameter would be more suitable.

Reply:
Now is changed, and don’t mind my typo, stupid autocorrect